.vscode/settings.json

```diff
@@ -19,7 +19,16 @@
 
   // Python analysis
   "python.analysis.typeCheckingMode": "off", // Disable Pylance strict checking
-  "python.analysis.exclude": ["**/*_pb2.py", "**/*_pb2.pyi"],
+  "python.analysis.exclude": [
+    // Default paths to exclude
+    "**/node_modules",
+    "**/__pycache__",
+    "**/.*",
```

Comment:
Currently, pylance checks `node_modules` and `__pycache__`, which significantly slows down code analysis.